tests/sealKey/enc/enc.cpp

```diff
@@ -142,11 +203,55 @@ bool TestOEGetSealKey()
         OE_TEST(second_key_buffer_size == sizeof(sgx_key_t));
         OE_TEST(key_info_size == sizeof(sgx_key_request_t));
 
+        uint8_t* second_key_buffer_ptr = NULL;
+        size_t second_key_buffer_ptr_size = 0;
+        uint8_t* key_info_ptr = NULL;
+        size_t key_info_ptr_size = 0;
+
+        // Get the seal key by policy and output the key info.
+        ret = oe_get_seal_key_by_policy_v2(
+            (oe_seal_policy_t)seal_policy,
+            &second_key_buffer_ptr,
+            &second_key_buffer_ptr_size,
+            &key_info_ptr,
+            &key_info_ptr_size);
+        if (ret != OE_OK)
+        {
+            return false;
+        }
+
+        oe_free_key(second_key_buffer_ptr, key_info_ptr);
+
+        OE_TEST(second_key_buffer_ptr_size == sizeof(sgx_key_t));
+        OE_TEST(key_info_ptr_size == sizeof(sgx_key_request_t));
+
+        second_key_buffer_ptr = NULL;
+        key_info_ptr = NULL;
+        second_key_buffer_size = 0;
+        key_info_ptr_size = NULL;
+
+        // Get the seal key by policy and output the key info.
+        ret = oe_get_seal_key_by_policy(
+            (oe_seal_policy_t)seal_policy,
+            &second_key_buffer_ptr,
+            &second_key_buffer_ptr_size,
+            &key_info_ptr,
+            &key_info_ptr_size);
+        if (ret != OE_OK)
+        {
+            return false;
+        }
+
+        oe_free_key(second_key_buffer_ptr, key_info_ptr);
+
+        OE_TEST(second_key_buffer_ptr_size == sizeof(sgx_key_t));
+        OE_TEST(key_info_ptr_size == sizeof(sgx_key_request_t));
+
         uint8_t third_key_buffer[sizeof(sgx_key_t)] = {0};
         size_t third_key_buffer_size = sizeof(third_key_buffer);
 
         // Get the seal key using saved key info.
-        ret = oe_get_seal_key(
+        ret = oe_get_seal_key_v1(
```

Comment:
These tests should also be done with the v2 API to ensure that they have the correct info.

Reply:
done